### ISSUE TITLE
Feature Request: Open offline channels.

### ISSUE THREAD

**Issue by Techwolf** [vinszent/gnome-twitch#244]
This is useful for schedule programming, open channel and wait for the stream to go live.
Also useful to chat with others before the stream goes live.

**Comment by vinszent** [comment]
Yeah this is a much need feature, that I'll be working on now. It kinda ties into VODS, showing the channel info page, etc so that's why I haven't done anything until now.
